widgets/common/src/deploymentsView/configuration.ts

```diff
@@ -39,7 +39,12 @@ export const sharedConfiguration: Stage.Types.WidgetConfigurationDefinition[] =
         default: deploymentsViewColumnIds.filter(columnId => columnId !== 'environmentType'),
         type: Stage.Basic.GenericField.MULTI_SELECT_LIST_TYPE
     },
-    Stage.GenericConfig.PAGE_SIZE_CONFIG(50),
+    {
+        ...Stage.GenericConfig.PAGE_SIZE_CONFIG(50),
+        hidden: false,
+        name: configurationT('pageSize.name'),
```

Comment:
What do you think about converting the name to follow https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/1875083446/UI+Translation+Files+Guidelines?

```suggestion
        name: tConfiguration('pageSize.name'),
```

Or would it be too big change not fitting this PR?

Reply:
Sure thing, I'll do it :slightly_smiling_face: Good catch

Reply:
Done :slightly_smiling_face: